map-to-db5MAIN.xsl

```diff
@@ -0,0 +1,222 @@
+<?xml version="1.0"?>
```

Comment:
Please just change the existing map-to-MAIN to work with DB5 instead. I don't see much use in converting to DocBook 4 when we can have 5. (I.e. I think Dietrich if can create DB5, there is no DB4 support needed.)